CRITICAL - 2024-10-11 07:21:21 --> Error: Class "App\Controllers\PetModel" not found
in APPPATH\Controllers\Workflow_Product_Image.php on line 63.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 07:30:00 --> mysqli_sql_exception: Duplicate entry '3' for key 'PRIMARY' in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(89): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 07:30:01 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '3' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(89): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:30:01 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '3' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'3\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'3\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(89): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:30:01 --> [Caused by] mysqli_sql_exception: Duplicate entry '3' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'3\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'3\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'3\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(89): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 07:33:20 --> mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY' in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(88): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 07:33:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(88): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:33:20 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'Test\', \'1\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'Test\', \'1\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(88): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:33:20 --> [Caused by] mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'Test\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'Test\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'Test\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(88): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 07:35:21 --> mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY' in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(88): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 07:35:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(88): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:35:21 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(88): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:35:21 --> [Caused by] mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(88): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 07:36:51 --> mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY' in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(91): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 07:36:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(91): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:36:51 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(91): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:36:51 --> [Caused by] mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(91): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 07:38:14 --> mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY' in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(94): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 07:38:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(94): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:38:14 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(94): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:38:14 --> [Caused by] mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(94): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 07:44:17 --> mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY' in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(93): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 07:44:17 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (:product_id:, :product_img_id:, :url:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(93): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:44:17 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (\'17\', \'1\', \'uploads/product/1728632657_b3bfedd6e5a3c93aba97.jpg\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (\'17\', \'1\', \'uploads/product/1728632657_b3bfedd6e5a3c93aba97.jpg\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (:product_id:, :product_img_id:, :url:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(93): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:44:17 --> [Caused by] mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (\'17\', \'1\', \'uploads/product/1728632657_b3bfedd6e5a3c93aba97.jpg\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (\'17\', \'1\', \'uploads/product/1728632657_b3bfedd6e5a3c93aba97.jpg\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (\'17\', \'1\', \'uploads/product/1728632657_b3bfedd6e5a3c93aba97.jpg\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (:product_id:, :product_img_id:, :url:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(93): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 07:46:52 --> mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY' in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(93): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 07:46:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(93): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:46:52 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(93): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:46:52 --> [Caused by] mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'17\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(93): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 07:48:33 --> mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY' in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(95): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 07:48:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(95): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:48:33 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(95): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:48:33 --> [Caused by] mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(95): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 07:50:43 --> mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY' in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(95): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 07:50:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(95): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:50:43 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(95): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 07:50:43 --> [Caused by] mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(95): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 10:05:31 --> mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY' in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(95): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 10:05:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(95): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:05:31 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(95): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:05:31 --> [Caused by] mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(95): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:08:18 --> ErrorException: Undefined array key "url"
in APPPATH\Controllers\Workflow_Product_Image.php on line 97.
 1 APPPATH\Controllers\Workflow_Product_Image.php(97): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "url"', 'C:\\xampp\\htdocs\\projects\\petzadmin\\app\\Controllers\\Workflow_Product_Image.php', 97)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:08:53 --> ErrorException: Undefined array key "url"
in APPPATH\Controllers\Workflow_Product_Image.php on line 96.
 1 APPPATH\Controllers\Workflow_Product_Image.php(96): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "url"', 'C:\\xampp\\htdocs\\projects\\petzadmin\\app\\Controllers\\Workflow_Product_Image.php', 96)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:09:31 --> ErrorException: Undefined array key "url"
in APPPATH\Controllers\Workflow_Product_Image.php on line 96.
 1 APPPATH\Controllers\Workflow_Product_Image.php(96): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "url"', 'C:\\xampp\\htdocs\\projects\\petzadmin\\app\\Controllers\\Workflow_Product_Image.php', 96)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:10:06 --> ErrorException: Undefined array key "url"
in APPPATH\Controllers\Workflow_Product_Image.php on line 96.
 1 APPPATH\Controllers\Workflow_Product_Image.php(96): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "url"', 'C:\\xampp\\htdocs\\projects\\petzadmin\\app\\Controllers\\Workflow_Product_Image.php', 96)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:11:39 --> ErrorException: Undefined array key "url"
in APPPATH\Controllers\Workflow_Product_Image.php on line 70.
 1 APPPATH\Controllers\Workflow_Product_Image.php(70): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "url"', 'C:\\xampp\\htdocs\\projects\\petzadmin\\app\\Controllers\\Workflow_Product_Image.php', 70)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:11:55 --> ErrorException: Undefined array key "url"
in APPPATH\Controllers\Workflow_Product_Image.php on line 70.
 1 APPPATH\Controllers\Workflow_Product_Image.php(70): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "url"', 'C:\\xampp\\htdocs\\projects\\petzadmin\\app\\Controllers\\Workflow_Product_Image.php', 70)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 10:13:01 --> mysqli_sql_exception: Column 'url' cannot be null in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(100): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 10:13:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'url' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (:product_img_id:, :product_id:, :url:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(100): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:13:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'url' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (:product_img_id:, :product_id:, :url:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(100): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:13:02 --> [Caused by] mysqli_sql_exception: Column 'url' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (:product_img_id:, :product_id:, :url:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(100): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 10:13:27 --> mysqli_sql_exception: Column 'url' cannot be null in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(100): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 10:13:27 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'url' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (:product_img_id:, :product_id:, :url:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(100): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:13:27 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'url' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (:product_img_id:, :product_id:, :url:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(100): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:13:27 --> [Caused by] mysqli_sql_exception: Column 'url' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (:product_img_id:, :product_id:, :url:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(100): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 10:15:28 --> mysqli_sql_exception: Column 'url' cannot be null in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(154): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 10:15:28 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'url' cannot be null
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (:product_img_id:, :product_id:, :url:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(154): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:15:28 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'url' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (:product_img_id:, :product_id:, :url:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(154): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:15:28 --> [Caused by] mysqli_sql_exception: Column 'url' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (:product_img_id:, :product_id:, :url:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(154): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 10:20:58 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'petz24_db', 3306, '', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `product...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product.php(46): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product->getProduct()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `product...', Array, false)
#3 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product.php(46): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product->getProduct()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product))
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-10-11 10:20:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `product_tbl`.`product_id` as `id`, `product_tbl`.`pet_id` as `pet_id`, `product_tbl`.`brand_id` as `brand_id`, `product_tbl`.`breed_id` as `breed_id`, `product_tbl`.`product_type_id` as `product_type_id`, `product_tbl`.`product_category_id` as `product_category_id`, `product_tbl`.`name` as `name`, `product_tbl`.`summery` as `summery`, `product_tbl`.`description` as `description`, `product_tbl`.`instruction` as `instruction`, `pet_tbl`.`pet_name` as `pet_name`, `brand_tbl`.`brand_name` as `brand_name`, `breed_tbl`.`breed_name` as `breed_name`, `product_type_tbl`.`type` as `type`, `product_category_tbl`.`category` as `category`
FROM `product_tbl`
INNER JOIN `pet_tbl` ON `product_tbl`.`pet_id` = `pet_tbl`.`pet_id`
INNER JOIN `product_type_tbl` ON `product_tbl`.`product_type_id` = `product_type_tbl`.`product_type_id`
INNER JOIN `product_category_tbl` ON `product_tbl`.`product_category_id` = `product_category_tbl`.`product_category_id`
INNER JOIN `brand_tbl` ON `product_tbl`.`brand_id` = `brand_tbl`.`brand_id`
INNER JOIN `breed_tbl` ON `product_tbl`.`breed_id` = `breed_tbl`.`breed_id`
WHERE `product_tbl`.`flag` = 1', [], false)
 3 APPPATH\Controllers\Workflow_Product.php(46): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product->getProduct()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 10:21:03 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'petz24_db', 3306, '', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(24): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->getProductImage()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(24): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->getProductImage()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-10-11 10:21:03 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_img_tbl`
JOIN `product_tbl` ON `product_tbl`.`product_id` = `product_img_tbl`.`product_id`
WHERE `product_img_tbl`.`flag` = 1', [], false)
 3 APPPATH\Controllers\Workflow_Product_Image.php(24): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->getProductImage()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 10:21:11 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'petz24_db', 3306, '', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(24): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->getProductImage()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(24): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->getProductImage()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-10-11 10:21:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product_img_tbl`
JOIN `product_tbl` ON `product_tbl`.`product_id` = `product_img_tbl`.`product_id`
WHERE `product_img_tbl`.`flag` = 1', [], false)
 3 APPPATH\Controllers\Workflow_Product_Image.php(24): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->getProductImage()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 10:21:15 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'petz24_db', 3306, '', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `product...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product.php(46): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product->getProduct()
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product))
#7 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `product...', Array, false)
#3 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product.php(46): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product->getProduct()
#5 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product))
#6 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-10-11 10:21:15 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `product_tbl`.`product_id` as `id`, `product_tbl`.`pet_id` as `pet_id`, `product_tbl`.`brand_id` as `brand_id`, `product_tbl`.`breed_id` as `breed_id`, `product_tbl`.`product_type_id` as `product_type_id`, `product_tbl`.`product_category_id` as `product_category_id`, `product_tbl`.`name` as `name`, `product_tbl`.`summery` as `summery`, `product_tbl`.`description` as `description`, `product_tbl`.`instruction` as `instruction`, `pet_tbl`.`pet_name` as `pet_name`, `brand_tbl`.`brand_name` as `brand_name`, `breed_tbl`.`breed_name` as `breed_name`, `product_type_tbl`.`type` as `type`, `product_category_tbl`.`category` as `category`
FROM `product_tbl`
INNER JOIN `pet_tbl` ON `product_tbl`.`pet_id` = `pet_tbl`.`pet_id`
INNER JOIN `product_type_tbl` ON `product_tbl`.`product_type_id` = `product_type_tbl`.`product_type_id`
INNER JOIN `product_category_tbl` ON `product_tbl`.`product_category_id` = `product_category_tbl`.`product_category_id`
INNER JOIN `brand_tbl` ON `product_tbl`.`brand_id` = `brand_tbl`.`brand_id`
INNER JOIN `breed_tbl` ON `product_tbl`.`breed_id` = `breed_tbl`.`breed_id`
WHERE `product_tbl`.`flag` = 1', [], false)
 3 APPPATH\Controllers\Workflow_Product.php(46): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product->getProduct()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 10:25:34 --> mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY' in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(149): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 10:25:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(149): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:25:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(149): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:25:34 --> [Caused by] mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (\'18\', \'2\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`) VALUES (:product_id:, :product_img_id:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(149): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:27:06 --> ErrorException: Undefined array key "url"
in APPPATH\Controllers\Workflow_Product_Image.php on line 151.
 1 APPPATH\Controllers\Workflow_Product_Image.php(151): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "url"', 'C:\\xampp\\htdocs\\projects\\petzadmin\\app\\Controllers\\Workflow_Product_Image.php', 151)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 10:29:52 --> mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY' in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(156): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 10:29:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (:product_id:, :product_img_id:, :url:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(156): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:29:52 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (\'18\', \'2\', \'uploads/product/1728630445_9f7361b110e02b27e2ca.jpg\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (\'18\', \'2\', \'uploads/product/1728630445_9f7361b110e02b27e2ca.jpg\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (:product_id:, :product_img_id:, :url:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(156): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:29:52 --> [Caused by] mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (\'18\', \'2\', \'uploads/product/1728630445_9f7361b110e02b27e2ca.jpg\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (\'18\', \'2\', \'uploads/product/1728630445_9f7361b110e02b27e2ca.jpg\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (\'18\', \'2\', \'uploads/product/1728630445_9f7361b110e02b27e2ca.jpg\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (:product_id:, :product_img_id:, :url:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(156): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 10:31:08 --> mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY' in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(156): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 10:31:08 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (:product_id:, :product_img_id:, :url:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(156): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:31:08 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (\'18\', \'2\', \'uploads/product/1728630445_9f7361b110e02b27e2ca.jpg\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (\'18\', \'2\', \'uploads/product/1728630445_9f7361b110e02b27e2ca.jpg\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (:product_id:, :product_img_id:, :url:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(156): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:31:08 --> [Caused by] mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (\'18\', \'2\', \'uploads/product/1728630445_9f7361b110e02b27e2ca.jpg\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (\'18\', \'2\', \'uploads/product/1728630445_9f7361b110e02b27e2ca.jpg\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (\'18\', \'2\', \'uploads/product/1728630445_9f7361b110e02b27e2ca.jpg\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_id`, `product_img_id`, `url`) VALUES (:product_id:, :product_img_id:, :url:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(156): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-10-11 10:32:47 --> mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY' in C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\xampp\htdocs\projects\petzadmin\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\projects\petzadmin\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\projects\petzadmin\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projects\petzadmin\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\projects\petzadmin\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\projects\petzadmin\app\Controllers\Workflow_Product_Image.php(154): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
#10 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
#11 C:\xampp\htdocs\projects\petzadmin\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\projects\petzadmin\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2024-10-11 10:32:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (:product_img_id:, :product_id:, :url:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Workflow_Product_Image.php(154): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
 8 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
 9 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:32:47 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', \'uploads/product/1728630445_9f7361b110e02b27e2ca.jpg\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', \'uploads/product/1728630445_9f7361b110e02b27e2ca.jpg\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (:product_img_id:, :product_id:, :url:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Workflow_Product_Image.php(154): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-10-11 10:32:47 --> [Caused by] mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', \'uploads/product/1728630445_9f7361b110e02b27e2ca.jpg\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', \'uploads/product/1728630445_9f7361b110e02b27e2ca.jpg\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (\'2\', \'18\', \'uploads/product/1728630445_9f7361b110e02b27e2ca.jpg\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `product_img_tbl` (`product_img_id`, `product_id`, `url`) VALUES (:product_img_id:, :product_id:, :url:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Workflow_Product_Image.php(154): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Workflow_Product_Image->updateProductImage()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Workflow_Product_Image))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
